static/js/place/place_template.tsx

```diff
@@ -751,9 +751,6 @@ class Chart extends Component<ChartPropType, ChartStateType> {
           for (const dataGroup of dataGroups) {
             dataGroup.label = STATS_VAR_TEXT[dataGroup.label];
           }
-          this.setState({
```

Comment:
why is this removed?

Reply:
Oops. It was removed by accident. Added it back.